idunn/geocoder/models/geocodejson.py

```diff
@@ -171,6 +171,7 @@ class IntentionDescription(BaseModel):
     query: Optional[str]
     category: Optional[str]
     place: Optional[Feature]
+    place_in_query: bool = False
```

Comment:
I'm not very fond of including this implementation detail in the API.  
Perhaps [`PrivateAttr`](https://pydantic-docs.helpmanual.io/usage/models/#private-model-attributes) could be used instead, although it would need to be set explicitly, and not via `__init__`.

Reply:
This sounds to fit perfectly the use case, maybe it could also be used for `type` ?

Reply:
As you wish. `type` looks well-defined enough. I don't mind keeping it that way.   
And using private attributes can be cumbersome as they can't be set by the default constructor (as far as I can see).